### ISSUE TITLE
'utf-8' codec can't decode byte 0xff in position 336: invalid start byte

### ISSUE THREAD

**Issue by SrikanthPusarla** [shap/shap#2764]
I am not able to load my Xgboost booster object into shap.TreeExplainer which is throwing error:  'utf-8' codec can't decode byte 0xff in position 336: invalid start byte.

But where as it is working good for the example provided at https://shap.readthedocs.io/en/latest/example_notebooks/tabular_examples/tree_based_models/Basic%20SHAP%20Interaction%20Value%20Example%20in%20XGBoost.html,

it seems shap.TreeExplainer is not working on booster build based on my parameters.

My code for training booster is below:

import xgboost as xgb
import shap
train = xgb.DMatrix(data=X_train, label=y_train, weight=Weights)
ym  = y_train.mean()
spw  = (ym / (1 - ym)) if (ym > (1 - ym)) else ((1 - ym) / ym)
MODEL_PARAMETERS = {
        'base_score':       ym,                'booster':             'gbtree', 'colsample_bylevel':       1,    
        'colsample_bynode': 1,                 'colsample_bytree':    0.5,      'gamma':                   0,    
        'gpu_id':           -1,                'importance_type':     'gain',   'interaction_constraints': '',   
        'learning_rate':    0.1,               'max_delta_step':      0,        'max_depth':               28,   
        'min_child_weight': 1,                 'missing':             np.nan,   'monotone_constraints':    '()', 
        'n_estimators':     164,               'n_jobs':              20,       'num_parallel_tree':       1,    
        'objective':        'binary:logistic', 'random_state':        0,        'reg_alpha':               0.0,  
        'reg_lambda':       0.00075,           'scale_pos_weight':    spw,      'subsample':               0.8,  
        'tree_method':      'approx',          'validate_parameters': 1,        'verbosity':               0     
        }
 
    booster = xgb.train(
      params=MODEL_PARAMETERS,
      dtrain=train,
      num_boost_round=NUM_BOOST_ROUNDS,
      evals=[(validation, "validation")], 
      early_stopping_rounds=EARLY_STOPPING_ROUNDS,
    )

explainer = shap.TreeExplainer(booster)

Python Environment:
xgboost: 1.5.2
shap: 0.40.0

**Comment by Moemenhussein11** [comment]
i have the same issue! did you figure out a solution?!
